README.md

```diff
@@ -1,2 +1,16 @@
-# substrate-stdlib
-A set of Move tools for the Substrate Framework which can be used in the MoveVM pallet.
+# Substrate-stdlib for Pallet Move
+
+This repository is part of [Pallet Move](https://github.com/eigerco/pallet-move), which enables users to write smart contracts in Substrate-based blockchains with the Move language. This standard library can be included to provide elementary Substrate functions in Move smart contracts.
+
+Currently, it cointains the module:
```

Comment:
```suggestion
Currently, it contains the module:
```

Reply:
Maybe the developer worked on some coin-example before :rofl:
Done :+1: